var/spack/repos/builtin/packages/gasnet/package.py

```diff
@@ -157,11 +157,34 @@ def test_install(self):
                 purpose="Running testtools",
```

Comment:
This `self.run_test()` invocation now generates a warning:
```
==> [2023-05-22-15:15:05.630432] Warning: The 'run_test' method is deprecated. Use 'test_part' instead for gasnet to process testtools.
```
I think the fix is to replace this call with `self.test_testtools()`

Reply:
Great catch!  Thanks.  I need tomato sure I replace those as well.